lms/djangoapps/learner_dashboard/api/v0/views.py

```diff
@@ -372,13 +372,14 @@ def get(self, request):
         for course_id in course_keys:
             course_data = get_course_data(course_id)
             if course_data:
-                recommended_courses.append({
-                    'course_run_key': [run['key'] for run in course_data['course_runs']
-                                       if run['availability'] == 'Current'][-1],
-                    'title': str(course_data['title']),
-                    'logo_image_url': course_data['owners'][0]['logo_image_url'],
-                    'marketing_url': course_data.get('marketing_url')
-                })
+                course_run_keys = [run['key'] for run in course_data['course_runs'] if run['availability'] == 'Current']
```

Comment:
We should use both current and upcoming availability statuses here.

Reply:
ok. let me add `Upcoming` state too in condition.